pod/perlfunc.pod

```diff
@@ -4444,46 +4444,84 @@ Leading white space is ignored without warning, as too are any trailing
 non-digits, such as a decimal point (L<C<oct>|/oct EXPR> only handles
 non-negative integers, not negative integers or floating point).
 
-=item open FILEHANDLE,EXPR
-X<open> X<pipe> X<file, open> X<fopen>
-
 =item open FILEHANDLE,MODE,EXPR
```

Comment:
Rearranging this should be accompanied with a change of all `L<C<open>/open FILEHANDLE,EXPR>` to the new top heading, there's also a few in perlport.